.gitignore

```diff
@@ -2,6 +2,7 @@ nugets
 deploy
 build32
 binaries
+zip
```

Comment:
Why was this changed?

Reply:
To avoid including the /zip folder that gets created if you build in release mode